src/domain/profile/modal/editProfileModal.module.scss

```diff
@@ -0,0 +1,52 @@
+@import 'mixins';
+@import 'variables';
+@import 'layout';
+
+.email {
+  label {
+    font-weight: bold;
+  }
+  p {
+    margin-top: 0.25em;
```

Comment:
eii no, use $baseMargin

Reply:
Good point. Looks like the other places that have the same margin are using `$baseMargin * 0.25`- so I'll use that here.